SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -1824,3 +1827,103 @@ void USpatialNetDriver::DelayedSendDeleteEntityRequest(Worker_EntityId EntityId,
 		Sender->SendDeleteEntityRequest(EntityId);
 	}, Delay, false);
 }
+
+void USpatialNetDriver::HandleStartupOpQueueing(const TArray<Worker_OpList*>& InOpLists)
+{
+	if (InOpLists.Num() == 0)
+	{
+		return;
+	}
+
+	QueuedStartupOpLists.Append(InOpLists);
+	bIsReadyToStart = FindAndDispatchStartupOps(InOpLists);
+
+	if (!bIsReadyToStart)
+	{
+	    return;
```

Comment:
Will data be in a valid state if execution returns here in case bIsReadyToStart == false? As I understand - second time execution gets into this function - QueuedStartupOpLists will not be empty and some Ops may be marked for skipping again?

Reply:
Hi Alex, yep data will be valid.  The skipping is only intended for when we stop blocking and process all the queued ops.

Reply:
I see, so we may queue Ops over multiple frames?

Reply:
yes :)

Reply:
I saw the answer above :)